storm/src/main/storm/mesos/util/PrettyProtobuf.java

```diff
@@ -223,7 +223,7 @@ private static String setToString(Set set) {
           break;
         default:
           // If hit, then a new Resource Type needs to be handled here.
-          value = "Unrecognized Resource Type: `" + r.getType() + "'";
+          value = String.format("Unrecognized Resource Type: `%s'", r.getType());
```

Comment:
I know this is comment is not related to your change - but notice backtick instead of quote in `\`%s''`?